Description: SimpleCov removal
 Removing the simplecov portion from the test as the simplecov version needed
 for ice_cube is not yet packaged.
 .
 ruby-ice-cube (0.11.1-1) UNRELEASED; urgency=medium
 .
   * Initial release (Closes: #nnnn)
Author: Balasankar C <balasankarc@autistici.org>


--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -1,9 +1,9 @@
-begin
-  require 'simplecov'
-  SimpleCov.start
-rescue LoadError
+#begin
+#  require 'simplecov'
+#  SimpleCov.start
+#rescue LoadError
   # okay
-end
+#end
 
 require 'ice_cube'
 
